src/libs/ReportActionsUtils.ts

```diff
@@ -1099,6 +1193,9 @@ function isOldDotReportAction(action: ReportAction): boolean {
  * For now, we just concat all of the text elements of the message to create the full message.
  */
 function getMessageOfOldDotReportAction(reportAction: OnyxEntry<ReportAction>): string {
+    if (!Array.isArray(reportAction?.message)) {
```

Comment:
Is this also affected in phase 2? If it does then we cannot return ''. We should return message.text, right?

Reply:
You're right.